app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from typing import List
+
+
+class Animal:
+    alive: List["Animal"] = []
```

Comment:
```suggestion
    alive: List[Animal] = []
```

Reply:
For some reason, he does not want to accept "Animal" without legs